README.md

```diff
@@ -86,6 +85,19 @@ To run the test suite with test weights (only works in branch `solutions`) while
 
 **NOTE**: all of these tasks tasks take the `currentProject` setting into account, see above.
 
+### Building docker container images for grading
+
+To build a docker container image to upload to Coursera to use for grading, be sure to have the following prerequisites installed:
+
+ - [`docker`](https://docker.com) - A popular container system for Linux.
+ - [`courseraprogramming`](https://github.com/coursera/courseraprogramming) - A toolkit to help instructional teams build grading containers
+
+To dockerize an sbt project, copy the sample docker file in `docker/Dockerfile.sample` to `Dockerfile` at the root of your project. Also copy `docker/grade.sh.template` to the root of your project. Then run: `docker build -t $TAG_NAME .` (e.g. `docker build -t reactive .`) to build and tag your container image and tag it as `reactive`.
```

Comment:
Sorry this was a post in which I dropped a part of the sentence. 

We are using `reactive` in all commands and `reactive` is a particular project. `sbt-coursera` is used for many courses and we should use something like `<your-project-name>` as a placeholder for the project name.